spec.md

```diff
@@ -1422,8 +1437,21 @@ message Error {
       // terminated or deleted before retrying.
       NODE_DOES_NOT_EXIST = 6;
 
+      // Indicates that the specified `NodeID` is not allowed or
+      // understood by the Plugin. More human-readable information MAY
+      // be provided in the `error_description` field.
+      //
+      // Recovery behavior: Caller MUST fix the `NodeID` before
+      // retrying.
       INVALID_NODE_ID = 7;
+
       VOLUME_NOT_ATTACHED_TO_SPECIFIED_NODE = 8;
+
+      // Indicates that the `NodeID` is not set to match that returned
+      // by `GetNodeID`.
```

Comment:
Ah, good catch. Correct, this error code is only for those plugins which actually require the node ID. I think we decided that if a plugin requires node ID but the CO lost it, then administrator intervention is required. Does that sound right?